### ISSUE TITLE
'tng_loss' vs 'loss' in in training_step

### ISSUE THREAD

**Issue by expectopatronum** [williamFalcon/pytorch-lightning#188]
### Before asking:   
- [x] 1. search the issues.  
- [x] 2. search the docs.    

If you still can't find what you need:     
#### What is your question?    

**What is the difference between `tng_loss` and `loss` in in `training_step`?** 
It seems both are displayed in my progress bar although only `tng_loss` is part of `prog` (unless `loss` comes from somewhere else but I don't think I specified it somewhere). But `tng_loss` jumps around alot more. My guess is `tng_loss` (or whatever I specify in `prog` is per batch). If this is correct the main question is, what is `loss` in the output, is it the same as specified in `training_step` and if yes, why is it different from `tng_loss` in the displayed output and why is it displayed although it is not part of `prog`?

#### Code    

My `training_step` based on the the documentation example:

```
def training_step(model, data_batch, batch_nb):
    input, label = data_batch

    out = model.forward(input)
    loss = model.loss(out, label)

    output = {
        'loss': loss, # required
        'prog': {'tng_loss': loss, 'batch_nb': batch_nb} # optional
    }
    # return a dict
    return output
```

**Comment by williamFalcon** [comment]
loss and tng_loss are the same here. anything in the ‘prog’ key of the dict is for progress bar display only.
